doc/website/sidebars.js

```diff
@@ -296,6 +296,11 @@ module.exports = {
                     id: 'guides/orms/sequelize',
                     label: 'Sequelize'
                 },
+                {
+                    type: 'doc',
```

Comment:
you can add this above sequelize?
since sequelize and typeorm are both JS, and hibernate is Java.

Reply:
Good point, @hilakashai what should be the order of the ORMs?

Reply:
@a8m

Reply:
I moved it above as @ronenlu suggested, if we want to or-order let me know